packages/isomorphic/ripemd160/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@xrplf/isomorphic/ripemd160",
```

Comment:
Why are these subpackages?

Reply:
This is how to achieve multiple entrypoints in a npm package.  This ensures you are only getting the code you want to use for bundling.  This allow allows bundlers and node to know which isomorphic implementation should be used.

Reply:
Does bundling matter much for these submodules? They're all very small and use the same `internal` code.

Reply:
It allows each to have a clear correlation to the noble libs submodules.

Reply:
It's so you avoid pulling in 'crypto' and thus 'crypto-browserify' which @ckniffen put a heap of work into removing so the bundle size is smaller (per https://github.com/XRPLF/xrpl.js/pull/2273#issuecomment-1506865731)